### PR TITLE
Feature/BC-476 Update email address

### PR DESCRIPTION
**Pull request by gooliajulia**
This PR implements the update email verification flow by:
* setting up the skeleton of the Settings portal (including re-usable `SideMenu` component)
* New confirmation screen for users once an email update has been submitted. Includes re-send email logic
* Uses existing EmailVerify flow to incorporate update email verifications as well as existing sign up verification flow
* Updates Sign In screen to auto-populate email address if coming from email verification flow

### CODE REVIEW

File: src/screens/Auth/EmailUpdate/UpdateEmailConfirmation.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        p {
+            font-size: 20px;
+        }
+
+        .resend {
+            color: purple;
+            font-size: 16px;
+            margin-top: 50px;
+            cursor: pointer;
```
Comment (mendo94) [inline src/screens/Auth/EmailUpdate/UpdateEmailConfirmation.scss:26]: does it need alphabetical or no

File: src/screens/Auth/EmailUpdate/UpdateEmailConfirmation.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      currentUserEmail,
+      newEmail,
+      authUser,
+      navigate
+    )
+
+    const severity = response.status >= 400 ? 'error' : 'success'
+
+    dispatch(
+      createSnackBar({
```
Comment (mendo94) [inline src/screens/Auth/EmailUpdate/UpdateEmailConfirmation.tsx:30]: I haven;t seen this done yet with useDispatch that is very neat!

**Review by mendo94 — DISMISSED** [review]
This is very clean, and was a great job!

File: src/screens/Auth/EmailUpdate/UpdateEmailConfirmation.tsx (hunk 2 above)
Comment (gooliajulia) [reply src/screens/Auth/EmailUpdate/UpdateEmailConfirmation.tsx:30]: This was all Koffi! Works great 🔥

**Review by mendo94 — APPROVED** [review]
merge with develop and you're golden

**Pull request merged**